reference/docs-conceptual/dev-cross-plat/vscode/using-vscode.md

```diff
@@ -204,60 +194,28 @@ text displayed in the session menu (also known as the `versionName`):
 }
 ```
 
-After you've configured this setting, restart Visual Studio Code or to reload the current Visual
-Studio Code window from the **Command Palette**, type **Developer: Reload Window**.
+After you've configured this setting, restart VS Code or to reload the current VS Code window from
+the **Command Palette**, type **Developer: Reload Window**.
```

Comment:
```suggestion
the **Command Palette**, type `Developer: Reload Window`.
```

I think that the "Developer:Reload Window" value should be code fenced since it's directly typed.